docs/integrations/sdk.md

```diff
@@ -10,12 +10,12 @@ React components for Lido Finance projects.
 - GitHub: [https://github.com/lidofinance/ui](https://github.com/lidofinance/ui)
 - NPM: [https://www.npmjs.com/search?q=%40lidofinance/](https://www.npmjs.com/search?q=%40lidofinance/)
 
-### Lido JS SDK
+### Lido Ethereum SDK
 
-Library for interaction with Lido contracts.
+Library for interaction with Lido protocol.
 
-- GitHub: [https://github.com/lidofinance/lido-js-sdk](https://github.com/lidofinance/lido-js-sdk)
-- NPM: [https://www.npmjs.com/search?q=%40lido-sdk/](https://www.npmjs.com/search?q=%40lido-sdk/)
+- GitHub: [https://github.com/lidofinance/lido-ethereum-sdk](https://github.com/lidofinance/lido-ethereum-sdk)
+- NPM: [https://www.npmjs.com/package/@lidofinance/lido-ethereum-sdk](https://www.npmjs.com/package/@lidofinance/lido-ethereum-sdk)
 
 ### Lido Python SDK
```

Comment:
Shall we deprecate the Python SDK?

Reply:
The better option might be to explicitly outline the target audience for each SDK/UI kit, wdyt, @DiRaiks?

Reply:
@TheDZhon, good suggestion!
I added `Target audience` for each sdk description